specification/cognitiveservices/data-plane/TranslatorText/readme.md

```diff
@@ -0,0 +1,120 @@
+# Cognitive Services Translator Text SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Translator Text SDK.
+
+The current release is `release_3_0`.
+
+``` yaml
+
+tag: release_3_0
+add-credentials: true
+openapi-type: data-plane
+```
+
+# Releases
+
+### Release 3.0
+These settings apply only when `--tag=release_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_3_0'
+input-file: v3.0/TranslatorText.json
```

Comment:
According to the short document in root page: https://github.com/Azure/azure-rest-api-specs
There should be either a `stable` folder or `preview` folder.
`stable/v3.0/TranslatorText.json`
`preview/v3.0/TranslatorText.json`